### PR TITLE
Update providers/devart/index.md

### PR DESCRIPTION
**Pull request by User235**

### CODE REVIEW

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/providers/devart/index.md](https://github.com/User235/EntityFramework.Docs/blob/patch-5/entity-framework/core/providers/devart/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C31%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201705310737330205-398%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: entity-framework/core/providers/devart/index.md [hunk 1]
```diff
@@ -48,4 +48,5 @@ See the [Devart dotConnect Entity Framework documentation](https://www.devart.co
 
 ## Supported Platforms
 
-* .NET Framework (4.5.1 onwards)
+* .NET Framework 2.0, 3.0, 3.5, 4.0, 4.5, 4.6, 4.7
```
Comment (bricelam) [inline entity-framework/core/providers/devart/index.md]: But EF Core itself doesn't support .NET Framework 2.0 thru 4.5, so how can this provider?

Comment (ErikEJ) [inline entity-framework/core/providers/devart/index.md]: Agree, this should not be changed...

**Comment by bradygaster** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** System.MissingMethodException: Method not found: &#39;Void Microsoft.DocAsCode.Plugins.DependencyType.set_Transitivity(Microsoft.DocAsCode.Plugins.DependencyTransitivity)&#39;.
   at Microsoft.DocAsCode.EntityModel.Plugins.OpenPublishing.SetDocumentId.GetDependencyTypesToRegister()
   at Microsoft.DocAsCode.Build.Engine.HostService.&lt;RegisterDependencyType&gt;b__81_0(IDocumentBuildStep buildStep)
   at Microsoft.DocAsCode.Build.Engine.BuildPhaseUtility.RunBuildSteps(IEnumerable`1 buildSteps, Action`1 action)
   at Microsoft.DocAsCode.Build.Engine.CompilePhaseHandlerWithIncremental.PreHandle(List`1 hostServices)
   at Microsoft.DocAsCode.Build.Engine.CompilePhaseHandlerWithIncremental.Handle(List`1 hostServices, Int32 maxParallelism)
   at Microsoft.DocAsCode.Build.Engine.PhaseProcessor.Process(List`1 hostServices, Int32 maxParallelism)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(PhaseProcessor phaseProcessor, List`1 hostServices, DocumentBuildContext context)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.BuildCore(DocumentBuildParameters parameter, IMarkdownServiceProvider markdownServiceProvider)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.Build(IList`1 parameters, String outputDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument(BuildJsonConfig config, TemplateManager templateManager, String baseDirectory, String outputDirectory, String pluginDirectory, String templateDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument()

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C1%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201706010736431137-398%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** System.MissingMethodException: Method not found: &#39;Void Microsoft.DocAsCode.Plugins.DependencyType.set_Transitivity(Microsoft.DocAsCode.Plugins.DependencyTransitivity)&#39;.
   at Microsoft.DocAsCode.EntityModel.Plugins.OpenPublishing.SetDocumentId.GetDependencyTypesToRegister()
   at Microsoft.DocAsCode.Build.Engine.HostService.&lt;RegisterDependencyType&gt;b__81_0(IDocumentBuildStep buildStep)
   at Microsoft.DocAsCode.Build.Engine.BuildPhaseUtility.RunBuildSteps(IEnumerable`1 buildSteps, Action`1 action)
   at Microsoft.DocAsCode.Build.Engine.CompilePhaseHandlerWithIncremental.PreHandle(List`1 hostServices)
   at Microsoft.DocAsCode.Build.Engine.CompilePhaseHandlerWithIncremental.Handle(List`1 hostServices, Int32 maxParallelism)
   at Microsoft.DocAsCode.Build.Engine.PhaseProcessor.Process(List`1 hostServices, Int32 maxParallelism)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(PhaseProcessor phaseProcessor, List`1 hostServices, DocumentBuildContext context)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.BuildCore(DocumentBuildParameters parameter, IMarkdownServiceProvider markdownServiceProvider)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.Build(IList`1 parameters, String outputDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument(BuildJsonConfig config, TemplateManager templateManager, String baseDirectory, String outputDirectory, String pluginDirectory, String templateDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument()

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C1%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201706010737113191-398%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by divega** [comment]
Superseded by #412.

**Pull request closed without merging**